grug/ffi/src/exports.rs

```diff
@@ -35,7 +35,7 @@ pub fn do_instantiate<M, E>(
 ) -> usize
 where
     M: DeserializeOwned,
-    E: Display,
+    E: Debug,
```

Comment:
@Rhaki can you revert the changes related to error debug here, because it's not the concern of this PR.